tms_django/apps/tms/tests/test_models.py

```diff
@@ -0,0 +1,67 @@
+from apps.tms.models import BookedLoad, Broker, DispatcherProfile, DriverProfile, Truck
+from apps.tms.tests.factories import (
+    BookedLoadFactory,
+    BrokerFactory,
+    DispatcherProfileFactory,
+    DriverProfileFactory,
+    TruckFactory,
+)
+from apps.users.models import User
+from django.test import TestCase
+
+
+class TestTruckModel(TestCase):
+    model_cls = Truck
+
+    def test_factory(self):
+        instance = TruckFactory()
+
+        instance.refresh_from_db()
+        self.assertIsInstance(instance, self.model_cls)
+        self.assertIsNotNone(instance.id)
+
+
+class TestDriverModel(TestCase):
+    model_cls = DriverProfile
+
+    def test_factory(self):
+        instance = DriverProfileFactory()
+
+        instance.refresh_from_db()
+        self.assertTrue(instance, self.model_cls)
+        self.assertTrue(isinstance(instance.user, User))
+        self.assertTrue(isinstance(instance.truck, Truck))
+
+
+class TestDispatcherProfileModel(TestCase):
+    model_cls = DispatcherProfile
+
+    def test_factory(self):
+        instance = DispatcherProfileFactory()
+
+        instance.refresh_from_db()
+        self.assertTrue(isinstance(instance, self.model_cls))
```

Comment:
check what kind of assert is used in TestTruckModel

Reply:
I don't understand your request. Please explain it to me

Reply:
@pyinto I cannot. I added 

```
class TestTruckModel(TestCase):
    model_cls = Truck

    def test_factory(self):
        instance = TruckFactory()

        instance.refresh_from_db()
        self.assertIsInstance(isinstance(instance, self.model_cls))
        self.assertIsNotNone(instance.id)
```

and I received this error. 

self.assertIsInstance(isinstance(instance, self.model_cls))
TypeError: TestCase.assertIsInstance() missing 1 required positional argument: 'cls'

GPT says to take off isinstance

Reply:
try to explain me what this row does:
`self.assertIsInstance(instance, self.model_cls)`

and then same for this one:
`self.assertIsInstance(isinstance(instance, self.model_cls))`

Reply:
I don't know it is a kind to check if this testing info are instance I guess 
The isinstance() function in Python is used to check if an object is an instance of a specified class or any of its subclasses